resources/admins.go

```diff
@@ -0,0 +1,16 @@
+package resources
+
+import (
+	"github.com/cloudquery/plugin-sdk/schema"
+	"github.com/cloudquery/plugin-sdk/transformers"
+	"github.com/onfleet/cq-source-onfleet/internal/go-onfleet/onfleet"
```

Comment:
Again, we recommend using our official wrapper that'll be released soon

Reply:
Answered above